facilitator/src/bin/facilitator.rs

```diff
@@ -547,260 +547,256 @@ fn main() -> Result<(), anyhow::Error> {
+
+    generate_ingestion_sample(
+        &mut *peer_transport,
+        &mut *own_transport,
+        &sub_matches
+            .value_of("batch-id")
+            .map_or_else(Uuid::new_v4, |v| Uuid::parse_str(v).unwrap()),
+        &sub_matches.value_of("aggregation-id").unwrap(),
+        &sub_matches.value_of("date").map_or_else(
+            || Utc::now().naive_utc(),
+            |v| NaiveDateTime::parse_from_str(&v, DATE_FORMAT).unwrap(),
+        ),
+        &PrivateKey::from_base64(sub_matches.value_of("pha-ecies-private-key").unwrap()).unwrap(),
+        &PrivateKey::from_base64(
+            sub_matches
+                .value_of("facilitator-ecies-private-key")
                 .unwrap(),
-                &ingestor_batch_signing_key,
-                sub_matches
-                    .value_of("dimension")
-                    .unwrap()
-                    .parse::<i32>()
-                    .unwrap(),
-                sub_matches
-                    .value_of("packet-count")
-                    .unwrap()
-                    .parse::<usize>()
-                    .unwrap(),
-                sub_matches
-                    .value_of("epsilon")
-                    .unwrap()
-                    .parse::<f64>()
-                    .unwrap(),
-                sub_matches
-                    .value_of("batch-start-time")
-                    .unwrap()
-                    .parse::<i64>()
-                    .unwrap(),
-                sub_matches
-                    .value_of("batch-end-time")
-                    .unwrap()
-                    .parse::<i64>()
-                    .unwrap(),
-            )?;
-            Ok(())
+        )
+        .unwrap(),
+        &ingestor_batch_signing_key,
+        sub_matches
+            .value_of("dimension")
+            .unwrap()
+            .parse::<i32>()
+            .unwrap(),
+        sub_matches
+            .value_of("packet-count")
+            .unwrap()
+            .parse::<usize>()
+            .unwrap(),
+        sub_matches
+            .value_of("epsilon")
+            .unwrap()
+            .parse::<f64>()
+            .unwrap(),
+        sub_matches
+            .value_of("batch-start-time")
+            .unwrap()
+            .parse::<i64>()
+            .unwrap(),
+        sub_matches
+            .value_of("batch-end-time")
+            .unwrap()
+            .parse::<i64>()
+            .unwrap(),
+    )?;
+    Ok(())
+}
+
+fn intake_batch(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
+    let mut intake_transport = intake_transport_from_args(sub_matches)?;
+
+    // We need the bucket to which we will write validations for the
+    // peer data share processor, which can either be fetched from the
+    // peer manifest or provided directly via command line argument. We
+    // check the manifest argument first because peer-output will always
+    // have at least the default value ".".
```

Comment:
This comment is out of date now that we've removed all default values, though that's not what this PR is about.